services/web/server/src/simcore_service_webserver/studies_dispatcher/_constants.py

```diff
@@ -1,13 +1,16 @@
 from typing import Final
 
-# NOTE: MSG_* strings MUST be human readable messages
+# NOTE: MSG_$(ERROR_CODE_NAME) strings MUST be human readable messages
 
-MSG_PROJECT_NOT_FOUND: Final[str] = "Cannot find any study with ID '{project_id}'."
 
+MSG_PROJECT_NOT_FOUND: Final[str] = "Cannot find any study with ID '{project_id}'"
 
-MSG_PROJECT_NOT_PUBLISHED: Final[
+
+MSG_PROJECT_NOT_PUBLISHED: Final[str] = "Cannot find any study with ID '{project_id}'"
+
+MSG_PUBLIC_PROJECT_NOT_PUBLISHED: Final[
     str
-] = "Cannot find any published study with ID '{project_id}'"
+] = "Cannot find any public study with ID '{project_id}'.\n Login and try again."
```

Comment:
I assume this is the case where we want the study to be accessible to all users **without** login. So this error message handles the case where the box in the DB is not checked. 
If this is correct, I'd rephrase to:
"This study is not public. \n Please login and try again or contact support. \n"

Reply:
This error happens when the linked study does not exists or is not shared with everyone or is not public ... But cannot know (due to the query) which of those conditions failed. 

NOTE: all messages include at the end ``For further information please copy this message and contact support@....``